spec/concepts/reservations/use_cases/create_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Reservations::UseCases::Create do
+  describe '.call' do
+    subject(:reservation) { described_class.new(**params) }
+
+    let(:user) { create(:user) }
+    let(:seance) { create(:seance) }
+    let(:params) do
+      {
+        user: user,
+        seance_id: seance.id,
+        seats: seats
+      }
+    end
+    let(:seats) { [1, 2] }
+
+    it 'creates reservations' do
+      expect { reservation.call }.to change(Reservation, :count).by(1)
```

Comment:
```suggestion
      expect { create_reservation }.to change(Reservation, :count).by(1)
```
part 2
I wanted to show you some other syntax, maybe it will be handy for you.

Reply:
I named it 'reservation' as it also returns a reservation. Its responsibility however is creation so maybe 'create_reservation' is a better name.